content/blog/2016/2016-09-01-jenkins-world-contributor-summit.adoc

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: "Jenkins World Contributor Summit"
+tags:
+- event
+- JenkinsWorld
+author: rtyler
+---
+
+
+image:/images/conferences/Jenkins-World_125x125.png[Jenkins World, role=right]
+
+
+At previous Jenkins User Conferences we have hosted "Contributor SUmmits" to
+gather developers and power-users in one room to discuss specific areas of
+Jenkins, such as Scalability, Pipeline, etc. At this year's
+link:https://www.cloudbees.com/jenkinsworld/home[Jenkins World] we're planning
+on discussing: link:/projects/blueocean[*Blue Ocean*],
+link:/doc/pipeline[*Pipeline*] and *Storage Pluggability*.
+
+
+Contributors to these three areas of the Jenkins ecosystem will be in
+attendance to present details on their design, requirements and tentative
+roadmaps. After the presentations, the afternoon is "unconference style" which
+is much more fluid to allow discussions, feedback, and brain-storming around
+the three focus areas.
+
+The program for the
+link:http://www.meetup.com/jenkinsmeetup/events/232811529/[*Jenkins World
+Contributor Summit*] includes:
+
+* Updates from the various project
+  link:https://wiki.jenkins-ci.org/display/JENKINS/Team+Leads[officers].
+* A discussion of the link:/projects/blueocean[Blue Ocean] technology stack,
+  overall architecture, and how to develop plugins that integrate with Blue
+  Ocean. Led by link:https://github.com/kzantow[Keith Zantow].
+* Presentation on the current status of link:/doc/pipeline[Pipeline], lessons
+  learned, new changes and the future. Led by
+  link:https://github.com/jglick[Jesse Glick].
+* Overview of "Storage Pluggability", a new scalability-oriented project to
+  revamp the underlying storage mechanisms in Jenkins. Led by
+  link:https://github.com/kohsuke[Kohsuke Kawaguchi].
+
+
+I cannot recommend participating in the Contributor Summit enough. I have found
+previous Summits to be immensely useful for sharing my own thoughts but also
+hearing new perspectives from the others in attendance.
+
+Our space is limited however! I encourage you to join us, so please
+link:http://www.meetup.com/jenkinsmeetup/events/232811529/[RSVP soon]!
+
+
+[CAUTION]
```

Comment:
IMPORTANT is a very different glyph in the CSS (:exclamation:), whereas CAUTION is a cute little :fire: